src/styles/blocks/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  max-width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    align-self: center;
+    display: block;
+  }
+
+  &__title {
+    color: $main-color;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 40px auto 0;
+  }
+
+  &__product {
+    line-height: 14px;
+    font-size: 10px;
+    color: $secondary-color;
```

Comment:
The variable `$secondary-color` is used here. Ensure that this variable is defined in your SCSS files. If it's not defined, it will cause a compilation error.